physics-informed-neural-networks-for-steady-cavity-flow/README.md

```diff
@@ -0,0 +1,274 @@
+
+# Cavity flow with Physics\-Informed Neural Networks
+
+Solve cavity flow governed by 2d steady Navier\-Stokes equations and continuity equation, using a Physics\-Informed Neural Network (PINN).
+
+
+The 2d, steady Navier\-Stokes equations for an incompressible fluid are:
+
+ $$ \frac{\partial u}{\partial x}+\frac{\partial v}{\partial y}=0 $$ 
+
+ $$ u\frac{\partial u}{\partial x}+v\frac{\partial u}{\partial y}+\frac{\partial p}{\partial x}-\frac{1}{Re}\bigg(\frac{\partial^2 u}{\partial x^2 }+\frac{\partial^2 u}{\partial y^2 }\bigg)=0 $$ 
+
+ $$ u\frac{\partial v}{\partial x}+v\frac{\partial v}{\partial y}+\frac{\partial p}{\partial y}-\frac{1}{Re}\bigg(\frac{\partial^2 v}{\partial x^2 }+\frac{\partial^2 v}{\partial y^2 }\bigg)=0 $$ 
+
+`(x,y)` are the spatial coordinates, `(u,v)` is the fluid velocity, `p` is the pressure, `Re` is the Reynolds number.
+
+
+In order to automatically satisfy the continuity equation we use the stream function psi such that `u=psi_y` and `v=-psi_x`. The boundary conditions are `(u,v)=(1,0)` and the top boundary and `(u,v)=(0,0)` at the other boundaries. The Reynolds number `Re=100`.
```

Comment:
It's nice to use LaTeX for $\psi$ and $u = \psi_y$ etc. or monospace `psi` if referring to a variable.

Reply:
I think using LaTeX in the text is a nice way to go -- I'll do that.